mobile-widgets/qml/main.qml

```diff
@@ -161,7 +161,6 @@ Kirigami.ApplicationWindow {
 	}
 
 	globalDrawer: Kirigami.GlobalDrawer {
-		id: gDrawer
```

Comment:
I agree in changing the id, but please do not just delete the line, change it to "id: globalDrawer" so we do not depend on the default id

Reply:
ok